packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.scss

```diff
@@ -55,13 +55,44 @@ $ProgressIndicatorTextHeight: 18px;
   transition: width .3s ease;
   width: 0;
 
+  &.indeterminate {
+    position: absolute;
```

Comment:
position: absolute is defined twice, not needed in both line 54 and 59

Reply:
Also recommend avoiding nesting. It creates very specific rules that are 2 levels deep, which creates problems later if it ever is overridden.